net/gallery-dl/Portfile

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           python 1.0
+
+github.setup        mikf gallery-dl 1.19.3 v
+github.tarball_from releases
+distname            gallery_dl-${github.version}
+
+categories          net
+maintainers         nomaintainer
+revision            0
+
+checksums           rmd160  a13818aa36cced652611d66046bed81b65285637 \
+                    sha256  db3973a17f1074e4f4cb99b635b94c1926bdd549263e1df909498f3c6aa93484 \
+                    size    389787
+
+description         command-line program to download image galleries and collections from several image hosting sites
+long_description    ${name} is a {*}${description} It is a cross-platform tool with many configuration options and powerful filenaming capabilities.
+
+platforms           darwin
+supported_archs     noarch
+license             GPL-2
+depends_lib         port:py${python.version}-requests
```

Comment:
please do specify `python.default_version  39` before the dependencies; this will make sure that the Python version doesn't suddenly change when it's updated in the `python` PG. One should first make sure that this port will build with Python 3.10 (the next default) before making that change automatically.